pkg/reconciler/events/build/controller.go

```diff
@@ -62,7 +64,8 @@ func NewController(
 	impl := cloudbuildsourcereconciler.NewImpl(ctx, r)
 
 	r.Logger.Info("Setting up event handlers")
-	cloudbuildsourceInformer.Informer().AddEventHandler(controller.HandleAll(impl.Enqueue))
+	cloudbuildsourceInformer.Informer().AddEventHandlerWithResyncPeriod(
```

Comment:
I'm surprised this is needed, I don't see it eventing at all. Are we sure there is nothing else in the former or the controller that is doing the resync?

Reply:
Talked over IM. Sounds like this is adding a 5 min re-reconcile, as opposed to the controller's more standard 10 hours. 

If we ever find this is reconciling too often, we can alter the time as appropriate.